src/benchmark/tpcds_benchmark.cpp

```diff
@@ -0,0 +1,172 @@
+#include <boost/algorithm/string.hpp>
+#include <boost/lexical_cast.hpp>
+
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include <random>
+#include <string>
+
+#include "SQLParser.h"
+#include "SQLParserResult.h"
+#include "benchmark_runner.hpp"
+#include "cli_config_parser.hpp"
+#include "cxxopts.hpp"
+#include "file_based_benchmark_item_runner.hpp"
+#include "file_based_table_generator.hpp"
+#include "json.hpp"
+#include "scheduler/current_scheduler.hpp"
+#include "scheduler/node_queue_scheduler.hpp"
+#include "scheduler/topology.hpp"
+#include "sql/sql_pipeline.hpp"
+#include "sql/sql_pipeline_builder.hpp"
+#include "sqlite_add_indices.hpp"
+#include "storage/chunk_encoder.hpp"
+#include "utils/assert.hpp"
+#include "visualization/lqp_visualizer.hpp"
+#include "visualization/pqp_visualizer.hpp"
+
+using namespace opossum;  // NOLINT
+
+namespace {
+
+bool data_files_available(const std::string& table_path);
```

Comment:
why don't you just but those functions here? It's quite common for main() to be at the bottom

Reply:
but = put?